src/tests/ftest/util/pool_create_all_base.py

```diff
@@ -44,10 +44,13 @@ def create_pool(self, index):
 
         return end_time - start_time
 
-    def create_one_pool(self):
+    # pylint: disable=dangerous-default-value
+    def create_one_pool(self, ranks=[]):
         """Create one pool with all the available storage capacity"""
         self.add_pool_qty(1, namespace="/run/pool/*", create=False)
-        self.pool[0].size.update("100%")
+        self.pool[0].size.update("100%", "pool[0].size")
+        if len(ranks) > 0:
```

Comment:
... Or, you could keep `ranks=None` in the param list and do this
```suggestion
        if ranks and len(ranks) > 0:
```

Reply:
Fixed.